bundles/org.openhab.binding.teleinfo/src/main/resources/OH-INF/thing/common-channel-types.xml

```diff
@@ -19,11 +19,11 @@
 		<category>Energy</category>
 		<state pattern="%s" readOnly="true">
 			<options>
-				<option value="TH">TH</option>
-				<option value="HC">HC</option>
-				<option value="HP">HP</option>
-				<option value="HN">HN</option>
-				<option value="PM">PM</option>
+				<option value="TH..">TH</option>
+				<option value="HC..">HC</option>
+				<option value="HP..">HP</option>
+				<option value="HN..">HN</option>
+				<option value="PM..">PM</option>
```

Comment:
Why change this?

Reply:
In the teleinfo protocol, the `PTEC` field values are coded on 4 characters. Unused characters are filled with dots.
Previously this field values were converted to an enum value and converted again to a string before being sent to channel state. The dots were removed in this process.
Since I remove the enum and pass directly the received value to channel state, the dots are now visible in the channel state. Would you prefer to ensure backward compatibility by removing this dots?

Reply:
I'd prefer backward compatibility if possible.

Reply:
42ecfab21f should ensure backward compatibility.